feature/gribi/ate_tests/hierarchical_weight_resolution_test/README.md

```diff
@@ -13,24 +13,22 @@ Configure ATE and DUT:
 
 *   Create a non-default VRF (VRF-1) that includes DUT port-1.
 
-*   On DUT port-2 and ATE port-2 create 18 L3 sub-interfaces each with a /30
-    subnet as below:
+*   For DUT port-2 interface, create a subinterface with Index 0 and IPv4 address 192.0.2.5, not in a VLAN.
```

Comment:
Why are we adding this change? Why cant this be starting from unit1 and vlanid 1?

Also, the PR is missing descriptions

Reply:
@sachendras ,
   We will add description, This PR is based on request to enhance this test based on https://github.com/openconfig/featureprofiles/pull/1133 . I see there were changes made in readme after PR 1133 was published. Can you and  @xw-g pls confirm  the readme to implement ?

Reply:
@xw-g is there a need for the unit IDs to start from "0"? The updates were made in https://github.com/openconfig/featureprofiles/pull/1239 to ensure all implementations pass. 
As such, there is no business requirement to support VLANs. VLANs exist only for the tests to not carry complicated TE configs.

Reply:
Note that my updates of test plan(https://github.com/openconfig/featureprofiles/pull/1133) does not include this. 

@manan-patel I think might be because this PR base does not include #1239. Can you help rebase or merge from the main to get the latest head? Thanks.

Reply:
done